floe.gemspec

```diff
@@ -35,7 +36,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "more_core_extensions"
   spec.add_dependency "optimist", "~>3.0"
 
-  spec.add_development_dependency "manageiq-style"
-  spec.add_development_dependency "rspec"
-  spec.add_development_dependency "rubocop"
+  spec.add_development_dependency "manageiq-style" # rubocop:disable Gemspec/DevelopmentDependencies
+  spec.add_development_dependency "rspec"          # rubocop:disable Gemspec/DevelopmentDependencies
+  spec.add_development_dependency "rubocop"        # rubocop:disable Gemspec/DevelopmentDependencies
```

Comment:
@kbrock we could actually move these to the Gemfile, what do you think?

Reply:
I kinda like them where they are, but yea. I'll do that.

Reply:
I don't really have an opinion either way but if it is a new cop that we're not sure we want wr could vote in manageiq-style

Reply:
I moved to gemfile. easier to just go with the flow on this one